### PR TITLE
PR for Input Component

### PR DESCRIPTION
**Pull request by manojasarsa**
PR for Input Component

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+@import url("/docs/main.css");
+
+.input_container {
+    padding: 2rem;
+    margin: 1rem 0 3rem 0;
+    border: 1px solid var(--light-grey-color);
+    box-shadow: 2px 2px 1px var(--shadow-color);
```
Comment (mimansha-swarup) [inline components/input/input.css:7]: you can make variable for shadow like this
--Shadow-01 : 2px 2px 1px var(--shadow-color);
and use Shadow-01 variable

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=css&width=680&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=14.5px&lh=147%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522input_container%2520flex_column%2522%253E%250A%2509%253Clabel%2520class%253D%2522input_label%2522%253EEmail%253C%252Flabel%253E%250A%2509%253Cinput%2520class%253D%2522input_box%2522%2520type%253D%2522email%2522%2520placeholder%253D%2522example%2540xyz.com%2522%2520required%2520%252F%253E%250A%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%253Cdiv%2520class%253D%2522input_container%2520flex_column%2522%253E%250A%2509%253Clabel%2520class%253D%2522input_label%2522%253ETextarea%253C%252Flabel%253E%250A%2509%253Ctextarea%2520class%253D%2522input_box%2522%2520cols%253D%252230%2522%2520rows%253D%25223%2522%2520placeholder%253D%2522write%2520your%2520thoughts...%2522%253E%253C%252Ftextarea%253E%250A%253C%252Fdiv%253E"
+                    style="width: 899px; height: 266px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (mimansha-swarup) [inline components/input/input.html:103]: suggestion: make iframe more responsive

**Review by mimansha-swarup — COMMENTED** [review]
except all looka good

**Review by ajitpsakri — APPROVED** [review]

**Review by vaibhav18matere — APPROVED** [review]
looks clean and organized, good work !

**Pull request merged**